src/raw/unix.rs

```diff
@@ -394,8 +395,6 @@ impl Socket {
     }
 
     ///Sets I/O parameters of socket.
-    ///

```

Comment:
Well the thing here is that this function does not exist on Windows and that's why i specified which one is used (so that user could find out acceptable parameters) :)